blueprint-serde/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+#![cfg_attr(feature = "std", no_std)]
```

Comment:
Some top-level crate docs could be useful, with some examples and how to use them.